packages/webpack-build-scripts/webpack.config.base.js

```diff
@@ -77,7 +64,6 @@ if (IS_PRODUCTION) {
 const scssLoaders = [
     {
         loader: require.resolve("css-loader"),
-        options: { minimize: IS_PRODUCTION },
```

Comment:
you need to add back minification through cssnano + postcss

Reply:
done. look at that minified CSS!
https://23198-71939872-gh.circle-artifacts.com/0/home/circleci/project/packages/docs-app/dist/docs-app.css